plugins/acpi-ivrs/fu-plugin-acpi-ivrs.c

```diff
@@ -48,6 +48,7 @@ fu_plugin_acpi_ivrs_add_security_attrs(FuPlugin *plugin, FuSecurityAttrs *attrs)
 	}
 	if (!fu_acpi_ivrs_get_dma_remap(ivrs)) {
 		fwupd_security_attr_set_result(attr, FWUPD_SECURITY_ATTR_RESULT_NOT_ENABLED);
+		fwupd_security_attr_add_flag(attr, FWUPD_SECURITY_ATTR_FLAG_ACTION_OEM_FW_UPDATE);
```

Comment:
Same statement as the DMAR optin.  Lenovo ties it to "optimized BIOS defaults" IIRC